spec/factories/activities.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :activity do
+    actor
+    object { create(:user) }
+    event 'test'
+    notifier_type 'test_notifier'
```

Comment:
Yes, I'll check it in the future when actual notifier exists. And I think I need to modify this factory, too.